Revised_read_jma

```diff
@@ -0,0 +1,26 @@
+            # Mark if this is a shortened line - does not contain 30 and 50 kts
+            # wind radius - set those to -1 to mark them as missing
+            if len(data) < 8:
+                self.max_wind_radius[i] = -1
+            else:
+                data[7] = int(data[7]) - int(int(data[7])/10000)*10000
+                data[9] = int(data[9]) - int(int(data[9])/10000)*10000
+                #data[8] = (data[7] + int(data[8]))/2
+                #data[10] = (data[9] + int(data[10]))/2
+                self.radius_30kts[i] = units.convert(float(data[9]), 'nmi', 'km')
```

Comment:
I am interested in the idea of storing these values rather than keeping them locally.  Have you properly initialized the arrays in the `__init__` method though?

Reply:
And about storing the value, I convert data[7] and data[9] into correct format and store them in self.radius_30kts and self.radius_50kts. And I initialized the arrays in the __init__ method though.